bash/functions.sh

```diff
@@ -332,62 +315,3 @@ movtomp4() {
     echo 'mkvtomp4 Usage: "cd" to the directory where the mkv video files are located and run "mkvtomp4" (then go grab a coffee).'
```

Comment:
Several useful functions were removed (webmify, gifify, launchlamp) without explanation. If these aren't needed, consider mentioning in the PR description why they were removed.